website/blog/2022-09-28-analyst-to-ae.md

```diff
@@ -0,0 +1,178 @@
+
+So let’s say that you have a business question, you have the raw data in your <Term id="data-warehouse"/>, and you’ve got dbt up and running. You’re in the perfect position to get this curated dataset completed quickly! Or are you?
+
+<!--truncate-->
+
+You’ve got three steps that stand between you and your finished curated dataset. *If you don’t have an Analytics Engineer*, then the work may be split up like this:
+
+![The in series development workflow](/img/blog/2022-09-28-analyst-to-ae/old_workflow.png)
+
+The first step is that your Analyst will start researching the stakeholder’s request and will try to boil it down to a high impact question that will drive the business. What do they really want to know? For example, your stakeholder might come to you with the following:
+
+***“We need to be able to track usage of our product, and we’d like to have some data around Active Users.”***
+
+Analysts are experts at taking broad statements and turning them into specific pieces of work.
+
+“Some data” could mean:
+
+1. A single KPI with a trendline over time
+1. A dashboard broken out by various categories
+1. A filterable and queryable table for ad-hoc analysis
+
+“Active Users” could mean:
+
+1. Users that logged in during a fixed time period
+1. Users with a session longer than a certain amount of time
+1. Users that interacted with a particular feature
+
+From there, the Analyst will build out the initial documentation and write down what they want the final dataset to look like. If your Analyst is not trained as an Analytics Engineer, this is the point that they will need to hand the project over to a Data Engineer to build the [model](https://docs.getdbt.com/docs/building-a-dbt-project/building-models).
+
+The first time that a team of Analysts and Data Engineers build a curated dataset, they will often expect this process to be a straight line to completion. Expectations are that the process will look something like this:
+
+![The expectation](/img/blog/2022-09-28-analyst-to-ae/first_loop.png)
+
+What quickly becomes apparent is that this push for a linear process often ends up creating three unanticipated “loops”:
+
+![The reality](/img/blog/2022-09-28-analyst-to-ae/multiple_loops.png)
+
+## Loop 1: Reacting to findings in the raw data
+
+Let’s say that your Analyst has defined the business need as an `Active Users` dataset with “Unique users that login during a given day”. The Analyst will try to do as much discovery work up-front as possible, because it’s hard to predict exactly what you’ll find in the raw data. When a Data Engineer gets stuck writing a model, they’ll need to reach out to the Analyst for some additional discovery. When an Analyst-turned-Analytics-Engineer has a question while writing a model, they don’t need to wait to talk to anyone, and they can begin researching it immediately. Which brings us to our first point:
```

Comment:
```suggestion
Let’s say that your Analyst has defined the business need as an `Active Users` dataset with “Unique users that log in during a given day”. The Analyst will try to do as much discovery work up-front as possible because it’s hard to predict exactly what you’ll find in the raw data. When a Data Engineer gets stuck writing a model, they’ll need to reach out to the Analyst for some additional discovery. When an Analyst-turned-Analytics-Engineer has a question while writing a model, they don’t need to wait to talk to anyone, and they can begin researching it immediately. Which brings us to our first point:
```